app/misc/git.ts

```diff
@@ -495,6 +495,56 @@ function createBranch() {
   }
 }
 
+// search for tags
+function searchTag() {
+  Git.Repository.open(repoFullPath)
+    .then(function (repo) {
+      repo.getCurrentBranch()
+        .then(function () {
+          // grab the list of references - these could be branches or tags
+          return repo.getReferences(Git.Reference.TYPE.LISTALL);
+        }).then(function (refList) {
+          refList.sort();
+          bname = {};
+          tags = {};
+          for (let i = 0; i < refList.length; i++) {
```

Comment:
Theres a missing closing curly bracket for this for loop. My typescript compiler didn't like it

Reply:
should be fixed now!